cordapp/src/main/kotlin/net/corda/plugins/Utils.kt

```diff
@@ -13,8 +12,19 @@ import kotlin.math.max
  * Mimics the "project.ext" functionality in groovy which provides a direct
  * accessor to the "ext" extension (See: ExtraPropertiesExtension)
  */
-@Suppress("UNCHECKED_CAST")
-fun <T : Any> Project.ext(name: String): T = (extensions.findByName("ext") as ExtraPropertiesExtension).get(name) as T
+
```

Comment:
I don't understand this function. Are our current semantics that `corda_release_version` _must_ be defined in `rootProject`? Does this changes those semantics? It looks like it's searching in every project between here and the root, which raises the possibility that someone will specify different values en route :worried:.

When I mentioned the `quasar-utils` plugin, I was only asking whether `rootProject.hasProperty(...)` and `rootProject.property(...)` would suffice ... :grin: